spec/models/project_spec.rb

```diff
@@ -663,6 +663,38 @@
     it { is_expected.to eq([reward_01, reward_03]) }
   end
 
+  describe '#accept_contributions?' do
+    subject { project.accept_contributions? }
+
+    context 'when project is online' do
+      context 'and is not expired' do
+        context 'and is available for contribution' do
+          let(:project) { create(:project, :online, :not_expired, available_for_contribution: true) }
+
+          it { is_expected.to be_truthy }
```

Comment:
Mas o `available_for_contributions?` já é definido, pq o nome do boolean criado é `available_for_contributions`. Vou colocar assim, ok? `expect(project).to be_accept_contributions`